src/wp-admin/includes/class-wp-posts-list-table.php

```diff
@@ -1432,47 +1482,53 @@ public function inline_edit() {
 				<span class="input-text-wrap"><input type="text" name="post_name" value="" /></span>
 			</label>
 
-	<?php endif; // $bulk
-	endif; // post_type_supports title ?>
+				<?php
+	endif; // $bulk
+	endif; // post_type_supports title
```

Comment:
indentation

Reply:
Agreed, will add to the amends already made locally ready to update the PR.